cmd/release/cmd/upload.go

```diff
@@ -0,0 +1,54 @@
+package cmd
+
+import (
+	"context"
+	"errors"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/feature/s3/manager"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/rancher/ecm-distro-tools/release/rancher"
+	"github.com/rancher/ecm-distro-tools/repository"
+
```

Comment:
Remove new line. Let's keep 2 groups. 1 for stdlib and the other for 3rd party libs.

Reply:
following the structure that @briandowns mentioned, shouldn't we just use `goimports` or `gofmt` to handle the formatting?

Reply:
`goimports` doesn't work properly if there is a name conflict, so I had to manually add the packages to the file